### DISCUSSION TITLE
Proposal: A way for folks to find teammates

### DISCUSSION THREAD

**Discussion by levilindsey** [ludumdare/ludumdare#2095] [Ideas]
It's very common for folks to want to find a team to participate in. Or for a team to want to find a person with a particular skill for their team.

This is something that folks use the current ldjam.com feed for. But I think the very general-purpose nature of that feed makes it not very useful for finding teammates.

And folks could find teammates in the Ludum Dare Discord server, but that's maybe not very discoverable.

Maybe a relatively simple way to support this would be to add a tags system to the current feed. Users could then add a tag like "team-search" to their post, and others could then filter the feed based on tags.

**Comment by mikekasprzak** [comment]
Short answer is yes, we need something.

Tagging posts with a "looking for group" tag, and adding a separate feed is an interesting approach. :thinking:
